src/components/Modals/CreateItemModal/CreateItemModal.tsx

```diff
@@ -255,21 +331,61 @@ export default class CreateItemModal extends React.PureComponent<Props, State> {
                 ) : null}
               </Column>
               <Column className="data" grow={true}>
-                <Section>
-                  <Header sub>{t('create_item_modal.representation_label')}</Header>
-                  <Row>
-                    {this.renderRepresentation(BodyShapeType.UNISEX)}
-                    {this.renderRepresentation(BodyShapeType.MALE)}
-                    {this.renderRepresentation(BodyShapeType.FEMALE)}
-                  </Row>
-                </Section>
+                {isAddingRepresentation ? null : (
+                  <Section>
+                    <Header sub>{t('create_item_modal.representation_label')}</Header>
+                    <Row>
+                      {this.renderRepresentation(BodyShapeType.UNISEX)}
+                      {this.renderRepresentation(BodyShapeType.MALE)}
+                      {this.renderRepresentation(BodyShapeType.FEMALE)}
+                    </Row>
+                  </Section>
+                )}
                 {bodyShape ? (
-                  <Field
-                    className="name"
-                    label={t('create_item_modal.name_label')}
-                    value={name}
-                    onChange={(_event, props) => this.setState({ name: props.value.slice(0, MAX_NAME_LENGTH) })}
-                  />
+                  <>
+                    {bodyShape === BodyShapeType.UNISEX ? (
+                      <Field className="name" label={t('create_item_modal.name_label')} value={name} onChange={this.handleNameChange} />
+                    ) : (
+                      <>
+                        {isAddingRepresentation ? null : (
+                          <Section>
+                            <Header sub>{t('create_item_modal.existing_item')}</Header>
+                            <Row>
+                              <div
+                                className={`option ${isRepresentation === true ? 'active' : ''}`}
+                                onClick={() => this.setState({ isRepresentation: true })}
```

Comment:
I asked this somewhere else but why's this not a bound instance method like `this.handleToggleIsRepresentation`?

Reply:
done 👍